wadas_webserver/test/test_database.py

```diff
@@ -45,25 +53,171 @@ def test_get_cameras(database):
     assert isinstance(cameras[0], Camera)
 
 
+def test_get_known_animals(database):
+    known_animals = database.get_known_animals()
+    assert len(known_animals) > 0
```

Comment:
Maybe just:
    assert known_animals
should be enough.
Worst case:
     assert len(known_animals)

Same on below other similar cases.